mappings/net/minecraft/client/gui/widget/TextFieldWidget.mapping

```diff
@@ -40,40 +43,63 @@ CLASS net/minecraft/class_342 net/minecraft/client/gui/widget/TextFieldWidget
 	METHOD method_1851 hasBorder ()Z
 	METHOD method_1852 setText (Ljava/lang/String;)V
 		ARG 1 text
+	METHOD method_1853 getDistanceBetween (I)I
+		ARG 1 position1
```

Comment:
What is the distance between here?

Reply:
`getDistanceToCursor`

Reply:
Using Runemoro's suggestion instead of treating it as an overload.